packages/react/src/patterns/sections/CardSectionSimple/README.md

```diff
@@ -0,0 +1,85 @@
+# CardSectionSimple
```

Comment:
Aren't we using the english name for here?

Reply:
like this `Card Section — Simple`